fivox/uriHandler.cpp

```diff
@@ -209,16 +213,29 @@ class URIHandler::Impl
 
     std::string getDyeCurve() const { return _get( "dyecurve" ); }
 
-    float getResolution() const { return _get( "resolution", _resolution ); }
+    float getResolution() const
+    {
+        return _get( "resolution", getFunctorType() == FUNCTOR_DENSITY
+                                   ? 0.0625f : _resolution );
```

Comment:
Not really magic, it's the 16um/voxel default that makes sense for densities. Same magics as we have for the input ranges of the different sources.